Initial Stub Connector (#1)
confluentinc/kafka-connect-aws-cloudwatch: README.md

# Introduction

> This project provides connectors for Kafka Connect to read and write data to AWSCloudWatch.

Reviewer: ```suggestion
This project provides connectors for Kafka Connect to read data from AWSCloudWatch.
```
